.github/vale/Vocab/Technology/accept.txt

```diff
@@ -9,8 +9,10 @@ HTTP
 IPs?
 IPv[46]
 IPvlan
+JPEG
 MAC
 RPM
+PNG
```

Comment:
How do the `exceptions:` above and this file relate? Do we need things like SVG, XML, YAML as well?

Reply:
@thaJeztah They are signals to our language linter that "PNG" whilst not appearing to be a real word, is correct in this case. We tend to just add to the list as and when we need to.

Reply:
Are they configuring the same rules? (was curious what "exceptions" does vs "accept" and if they intersect in some way).